src/status_im2/common/scan_qr_code/view.cljs

```diff
@@ -0,0 +1,239 @@
+      {:size   :paragraph-1
+       :weight :medium
+       :style  style/enable-camera-access-header}
+      (i18n/label title-label-key)]
+     [quo/text
+      {:size   :paragraph-2
+       :weight :regular
+       :style  style/enable-camera-access-sub-text}
+      (i18n/label description-label-key)]
+     [quo/button
+      {:before              button-icon
+       :type                :primary
+       :size                32
+       :accessibility-label accessibility-label
+       :override-theme      :dark
+       :customization-color :blue
+       :on-press            on-press}
+      (i18n/label button-label)]]
+  
+      ))
+
+
+(defn- qr-scan-hole-area
+  [qr-view-finder]
+  [rn/view
+   {:style     style/qr-view-finder
+    :on-layout (fn [event]
+                 (let [layout      (js->clj (oops/oget event "nativeEvent.layout")
+                                            :keywordize-keys
+                                            true)
+                       view-finder (assoc layout :height (:width layout))]
+                   (reset! qr-view-finder view-finder)))}])
+(defn- border
+  [border1 border2 corner]
+  [rn/view {:style (style/border border1 border2 corner)}])
+
+(defn- border-tip
+  [{:keys [top bottom left right]}]
+  [rn/view
+   {:style (style/border-tip top bottom right left)}])
+
+(defn- viewfinder
+  [qr-view-finder]
+  (let [size (+ (:width qr-view-finder) 2)]
+    [:<>
+     [rn/view {:style (style/viewfinder-container qr-view-finder)}
+      [rn/view
+       {:style (style/qr-view-finder-container size)}
+       [rn/view
+        {:style style/view-finder-border-container}
+        [rn/view
+         [border :border-top-width :border-left-width :border-top-left-radius]
+         [border-tip {:right -1 :top 0}]
+         [border-tip {:left 0 :bottom -1}]]
+        [rn/view
+         [border :border-top-width :border-right-width :border-top-right-radius]
+         [border-tip {:right 0 :bottom -1}]
+         [border-tip {:left -1 :top 0}]]]
+       [rn/view {:flex-direction :row :justify-content :space-between}
+        [rn/view
+         [border :border-bottom-width :border-left-width :border-bottom-left-radius]
+         [border-tip {:right -1 :bottom 0}]
+         [border-tip {:left 0 :top -1}]]
+        [rn/view
+         [border :border-bottom-width :border-right-width :border-bottom-right-radius]
+         [border-tip {:right 0 :top -1}]
+         [border-tip {:left -1 :bottom 0}]]]
+       [quo/button
+        {:icon                      true
+         :type                      :blur-bg
+         :size                      32
+         :accessibility-label       :camera-flash
+         :override-background-color colors/neutral-80-opa-40
+         :style                     style/camera-flash-button}
+        :i/flashlight-off]]
+      [quo/text
+       {:size   :paragraph-2
+        :weight :regular
+        :style  style/viewfinder-text}
+       (i18n/label :t/ensure-qr-code-is-in-focus-to-scan)]]]))
```

Comment:
new line needed

Reply:
you should add a line break at line 150 where the function ends and where (defn render-camera begins
the same way u did at line 174